api/nakadi-event-bus-api.yaml

```diff
@@ -1558,6 +1628,20 @@ definitions:
 
   EventTypeSchema:
     properties:
+      version:
```

Comment:
A few things here:

- semver allows trailing information as well as the three digit scheme. Is that allowed here?

- semver treats 0.y.z different to 1.y.z; the former (pre-1) range is allowed to have breaking changes. Is that allowed here?

- is semver definitely adequate to express what's needed for server versioning?

(gr)